src/signals_notebook/common_types.py

```diff
@@ -331,14 +331,16 @@ class File(BaseModel):
     name: str
     content: bytes
     content_type: str
+    file_size: int
```

Comment:
It would be better to have a @property file_size, which is calculated dynamically.